tools/inference_engine/src/impl/nodes/matmul.cpp

```diff
@@ -42,7 +63,35 @@ void inference_engine::GpuMatMul::compile(GpuContext& ctx)
     build_options += " -DM=" + std::to_string(get_M());
     build_options += " -DK=" + std::to_string(get_K());
     build_options += " -DN=" + std::to_string(get_N());
-    kernel_ = ctx.create_kernel("matmul_ref", code_string, std::strlen(code_string), build_options.c_str(), INFERENCE_ENGINE_KERNEL_LANGUAGE_OCL);
+
+    if (!post_ops_.empty())
```

Comment:
[note] this kind of code will probably have to be separated into some utility, helper function, but it can be done in another PR

Reply:
yes, this is just to make some basic tests passing